client/src/components/LoginSession/LoginSessionContainer.js

```diff
@@ -34,27 +34,32 @@ function LoginSessionContainer(props) {
       id: props.ID,
       SecurityID: Config.get('SecurityID')
     })
-      .then(response => {
-        const failure = !response.success;
-        setFailed(failure);
-
-        if (failure) {
-          props.displayToastFailure(response.message);
-        } else {
-          props.displayToastSuccess(response.message);
+    .then(response => {
+      props.displayToastSuccess(response.message);
+    })
+    .catch(err => {
```

Comment:
When we have error response, we check if we can parse the body to a JSON object with a message key